docs/resources/aws_db_subnet_groups.md

```diff
@@ -0,0 +1,66 @@
+---
+title: About the aws_db_subnet_group Resource
+platform: aws
+---
+
+# aws\_db\_subnet\_group
+
+Use the `aws_db_subnet_group` InSpec audit resource to test properties of a collection of AWS RDS subnet groups.
+
+RDS gives you access to the capabilities of a MySQL, MariaDB, PostgreSQL, Microsoft SQL Server, Oracle, or Amazon Aurora database server.
+
+## Syntax
+
+ Ensure you have exactly 3 subnet groups
+
+    describe aws_db_subnet_groups do
+      its('db_subnet_group_name.count') { should cmp 3 }
+    end
+    
+#### Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on RDS](https://docs.aws.amazon.com/rds/?id=docs_gateway).
+
+## Properties
+
+|Property                     | Description|
+| ---                         | --- |
+|db\_subnet\_group\_name    | The name of the DB subnet group. |
+|db\_subnet\_group\_description    | Provides the description of the DB subnet group. | 
+|vpc\_id    | Provides the VpcId of the DB subnet group. |
+|subnet\_group\_status    | Provides the status of the DB subnet group. | 
+|subnets    | Contains a list of Subnet elements. |
+|db\_subnet\_group\_arn    | The Amazon Resource Name for the DB subnet group. |
+
+## Examples
+
+##### Ensure DB Subnet Group Name of a subnet group exists
+    describe aws_db_subnet_groups do
+      its('db_subnet_group_name') { should include 'subnet-group-name' }
+    end
+
+## Matchers
+
+For a full list of available matchers, please visit our [Universal Matchers page](https://www.inspec.io/docs/reference/matchers/).
+
+#### exist
```

Comment:
I think these examples should use `aws_db_subnet_groups` plural resource with a `where` method to filter the table. That would give users better idea of how to benefit from the underlying capability.